go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/coreos/ignition/v2
 go 1.17
 
 require (
-	cloud.google.com/go/compute v1.10.0
```

Comment:
This does not seem right, reaching out to @bgilbert

Reply:
Looks like this package was split into a separate subpackage.